sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/state/StateStoreCoordinatorSuite.scala

```diff
@@ -208,7 +208,7 @@ class StateStoreCoordinatorSuite extends SparkFunSuite with SharedSparkContext {
           if (badPartitions.contains(partitionId)) {
             assert(latestSnapshotVersion.getOrElse(0) == 0)
           } else {
-            assert(latestSnapshotVersion.get >= 0)
+            assert(latestSnapshotVersion.get > 0)
```

Comment:
can you confirm this won't cause flakiness and we are definitely uploading snapshot for the good partitions